Client/Frontend/Strings.swift

```diff
@@ -1005,19 +1005,19 @@ extension String {
                 value: "Welcome to an independent internet",
                 comment: "String used to describes the title of what Firefox is on the welcome onboarding page for current version in our Onboarding screens.")
             public static let Description = MZLocalizedString(
-                key: "Onboarding.Welcome.Description.v114",
+                key: "Onboarding.Welcome.Description.v120",
```

Comment:
Updating strings is complicated I'm not sure if we should follow these steps from our [wiki](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings#steps-1) for these cases. The string freeze is October 17 so we still have time let me investigate how it should be done

Reply:
Do you mean to add the NotUsed part? I can add it

Reply:
Its been updated @yoanarios pls let me know if I need to do any more updates